examples/config_file/endlessh-go.yml

```diff
@@ -0,0 +1,42 @@
+# Connection type. Possible values are tcp, tcp4, tcp6
+conn_type: "tcp"
+# Enable prometheus
+enable_prometheus: false
+# Supplier to obtain Geohash of IPs. Possible values are "off", "ip-api", "max-mind-db"
+geoip_supplier: "off"
+# SSH listening address
+host: "0.0.0.0"
+# Message millisecond delay
+interval_ms: 1000
+# Maximum banner line length
+line_length: 32
+# When logging hits line file:N, emit a stack trace
+log_backtrace_at: []
+# If non-empty, write log files in this directory
+log_dir: ""
```

Comment:
How important are these log related flag to you?

They are from libraries I used, so I don't have control over them. The library may add/remove flags.

I may or may not be able to change the behavior. It needs some study.

Reply:
the log_backtrace_at var is only for debugging purpose and i don't think it is appropriate for release.
log_dir could be usefull for binary use (can't find a use case where it's usefull for containers)